question24/style.css

```
@@ -0,0 +1,185 @@
+body,
+div,
+dl,
+dt,
+dd,
+ul,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+pre,
+code,
+form,
+fieldset,
+legend,
+input,
+textarea,
+p,
+blockquote,
+th,
+td {
+  margin: 0;
+  padding: 0;
+}
+* {
+    box-sizing: border-box;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+img {
+    max-width: 100%;
+}
+body {
+    height: 100vh;
+    position: relative;
+    background-color: #ededd1;
+}
+.loading {
+    width: 100px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+.wrapper {
+    box-sizing: border-box;
+    width: 100%;
+    max-width: 500px;
+    margin: auto;
+    background: #fff;
+    padding: 35px 5%;
+    border-radius: 10px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translateY(-50%) translateX(-50%);
+    box-shadow: 3px 6px 10px #999;
+}
+h1 {
+    text-align: center;
+    color: #000;
+    font-size: 30px;
+    margin-bottom: 30px;
+}
+h2 {
+    text-align: center;
+    font-size: 28px;
+    margin-bottom: 25px;
+}
+.modal p {
+    font-size: 15px;
+}
+ol ol {
+    list-style: upper-alpha;
+}
+li {
+    margin-bottom: 15px;
+    list-style: none;
+}
+.submit_button {
+    width: 100%;
+    background: #000;
+    font-size: 1rem;
+    line-height: 1.2;
+    font-weight: 300;
+    height: 60px;
+    color: #fff;
+    border-radius: 30px;
+}
+.submit_button:disabled {
+    background: #ccc;
+}
+.close_button {
+    width: 50px;
+    height: 50px;
+    background: #000;
+    font-size: 2.5rem;
+    line-height: 1.2;
+    font-weight: 300;
+    color: #fff;
+    border-radius: 30px;
+    position: absolute;
+    top: 0;
+    right: 0;
+    cursor: pointer;
+}
+input {
+    width: 100%;
+    border: 1px solid gray;
+    height: 45px;
+    border-radius: 7px;
+}
+.checkbox {
+    margin: 20px 0;
+}
+.checkbox input {
+    height: 25px;
+    width: 25px;
+    vertical-align: middle;
+    margin-right: 8px;
+}
+.modal {
+    max-width: 500px;
+    width: 100%;
+    position: relative;
+    margin: 0 auto;
+    z-index: 2;
+    top: 13%;
+}
+.modal_container {
+    background: #fff;
+    overflow: scroll;
+    height: 350px;
+    border-radius: 10px;
+    padding: 30px;
+}
```

Since the rounded corners are effective only in the upper left of the scroll,
Specify the rounded corners and screen size on the outside, erase with overflow: hidden ;, and specify width: 100% and height: 100% on the inside!

```suggestion
.modal {
    max-width: 500px;
    width: 100%;
    height: 350px;
    border-radius: 10px;
    position: relative;
    margin: 0 auto;
    z-index: 2;
    top: 13%;
    overflow: hidden;
}
.modal_container {
    width: 100%;
    height: 100%;
    background: #fff;
    overflow: scroll;
    padding: 30px;
}
```

---

Thank you for your review! I learned a lot about here!:satisfied: I have made the changes you mentioned so please confirm.

fixed commit : https://github.com/csakyo/homework/pull/38/commits/7ef49267c17b4fd61d4af8288a090350c2b8591e